apps/spruce/src/pages/waterfall/WaterfallFilters.tsx

```diff
@@ -32,7 +36,7 @@ export const WaterfallFilters: React.FC<WaterfallFiltersProps> = ({
 
 // Temporary - update styles as more filters are added.
 const FilterItem = styled.div`
-  width: 300px;
+  ${({ width }: { width: number }) => `width: ${width}%;`}
```

Comment:
I think `flex-basis` would be the preferred property here
```suggestion
  ${({ width }: { width: number }) => `flex-basis: ${width}%;`}
```